src/nodes/ImageView.vue

```diff
@@ -34,13 +34,15 @@
 				<transition name="fade">
 					<template v-if="!failed">
 						<div v-if="isMediaAttachment"
-							class="media"
-							@click="handleImageClick(src)">
+							class="media">
 							<div class="media__wrapper">
-								<img v-show="loaded"
-									:src="imageUrl"
-									class="image__main"
-									@load="onLoaded">
+								<a :href="internalLinkOrImage"
```

Comment:
I would put the `a` tag around the div, not inside it. So that the metadata div becomes part of the link.

Reply:
Will fix it.